src/components/Search/SearchBar/index.tsx

```diff
@@ -66,34 +62,26 @@ const SearchBar: React.FC<SearchBarProps> = ({
     { disabled, ignoreFocus: true }
   );
 
+  const debounceSearch = useRef(
+    debounce((value: string) => {
+      onChange(value);
+    }, 1000)
+  ).current;
```

Comment:
I'm confused about what this useRef is for?

Reply:
If its to maintain a stable reference to the debounce function you can use [`useCallback`](https://beta.reactjs.org/reference/react/useCallback)

Reply:
I needed a reference to `debounceSearch` to call the `cancel()` function on it, and I couldn't figure out how to do this with `useCallback` ☹️